cannon/mipsevm/singlethreaded/state.go

```diff
@@ -185,6 +185,15 @@ func (s *State) EncodeWitness() ([]byte, common.Hash) {
 	return out, stateHashFromWitness(out)
 }
 
+func (s *State) EncodeProof() []byte {
+	proofData := make([]byte, 0)
+	insnProof := s.Memory.MerkleProof(s.Cpu.PC)
+	proofData = append(proofData, insnProof[:]...)
+	//Fake SecondMemoryProof
+	proofData = append(proofData, insnProof[:]...)
+	return proofData
```

Comment:
We can't generate fake data in production code - if necessary, this should be a test utility.